public/caesar-shift/encrypt.py

```diff
@@ -0,0 +1,53 @@
+import string
+from encrypt_library import *
+#START#
+def caesar_shift(plaintext, shift_value=5):
+  ciphertext = ""
+  #<a href="#" class="clickable" class="clickable" onclick="if(enable_tooltips){$('#modalForLoop').modal('open')}">for eachletter in plaintext:</a> 
+  for eachletter in plaintext.lower():
+    #if our letter is not a letter, keep it and move on!
+    if eachletter not in string.ascii_letters:
+      ciphertext = ciphertext + eachletter
+      continue
+
+    #if adding shift constant puts out of alphabet (> ord(z)) then sub ord(a)
```

Comment:
Is this a relic? Looks similar to line 22; standardize them. Also, is this referring to what you're going to do in shift_by_value_in_alphabet? 

If so, it might be worth eliminating this. Put a good explanation in the docstring fro shift_by_value_in_alphabet(), and leave them with some way to see that, and just gloss over it, here.